src/python/serenity_types/pricing/derivatives/rates/yield_curve.py

```diff
@@ -174,6 +184,12 @@ class RawYieldCurve(BaseModel):
     A term structure of yield curve inputs. The RAW representation is offered to clients so they
     can either do their own interpolation or for diagnostics.
     """
+
+    rate_source_types: List[RateSourceType]
```

Comment:
From the code, seems like RawYieldCurve will be generated based on rate_source_types.
And same comment with above